### DISCUSSION TITLE
OpenAi KEY　获取问题，

### DISCUSSION THREAD

**Discussion by v479038280** [ChatGPTNextWeb/NextChat#2776] [General]
请问下现在除了官网还有什么渠道能获取到KEY吗

**Comment by ericjuice** [comment]
> 请问下现在除了官网还有什么渠道能获取到KEY吗

可以看看 openkey.cloud，算是另一种更便宜的国内购买渠道

**Comment by Yidadaa** [comment]
可以考虑我的中转服务：https://api.nextweb.fun
